scripts/variantstore/variant_annotations_table/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+Most of the other files are specific to where the VAT will live, like the project_id and dataset_name and the table_suffix which will name the VAT itself as vat_`table_suffix` as well as a GCP bucket location, the output_path, for the intermediary files and the VAT export in tsv form.
+
+The other inputs can be copied from where I’ve put several files (maybe we should find a better place?!? Since many are in my scratch dir for now—tho anything in my scratch dir is also living in github as it’s things like what BQ schema is needed)
+
+The provided example WDL indicates all of the needed inputs.
+
+
+
+
+
+
+
+Here’s a brief overview of the pipeline itself:
+
+MakeSubpopulationFiles grabs Lee’s ancestry file and keeps only the columns it needs. It also grabs the File of shard names / paths. Each of those paths is then used to kick off one of the next step which is a subworkflow of three tasks. There will be a subworkflow for each vcf shard.
+
+  ExtractAnAcAfFromVCF is the first step of the subworkflow and filters out bad sites, sites with too many alt alleles, duplicate variants and calculates the AC and AN values for each subpopulation
+  AnnotateVCF is the second step of the subworkflow and adds Nirvana annotations to each of the variants
+  PrepAnnotationJson is the final step and prepares the data to be loaded into a BQ table, and loads it into a GCP staging bucket, using the output_path
+
+BigQueryLoadJson creates BQ tables (two temporary and one that is the VAT) and then gets everything from the staging buckets (genes and vt) and loads it into the two temp tables in BQ. Those two tables are joined together to create the VAT.
+
+BigQuerySmokeTest checks that the number of variants in the new VAT is the same as the expected number from the many subworkflows. This will never fail the pipeline, but is helpful to validate results.
+
+BigQueryExportVat takes the VAT and exports it by chromosome into a series of tsv files and loads it into a GCP bucket in the directory created by the output_path
+
+
+
+
+
+
+Notes:
+
+When running this pipeline, currently I try to stay around 5k-10k shards—I have yet to successfully run more than that, but I think it’s more about nerves than anything else. At 20k shards, the shards do tend to step on each other if there are too many—not that anything fails or is wrong, but I’d rather get results for a run that I then do multiple times than wait for queued jobs.
+
+
+The VAT table is created by that query fresh each time so that there is no risk of duplicates.
+HOWEVER the Genes and VT tables are not. They are cleaned up after 24 hours, but this code needs to be tweaked so that you can’t get into a state where duplicates are created here. The real question here is going to be, is there a use case that we might want to run where adding to a VAT that was created say weeks ago is beneficial, but given that calculations
+
+Note that there are two temporary tables that are created in addition to the main VAT table
```

Comment:
If you are going to include this, I would list the tables (and maybe which workflow creates them), too, but I'm not sure why this needs to be documented if they clean up themselves?  Or is there a scenario where someone might need to manually clean them up?